### DISCUSSION TITLE
ONVIF compatiblity

### DISCUSSION THREAD

**Discussion by Dvalin21** [roflcoopter/viseron#554] [Q&A]
So all my cameras are ONVIF compatible. 
I noticed last nice that detection was triggered, but it also triggered a function that was enabled in the camera app. The flood lights came on. Is it possible trigger other ONVIF functions or was it even suppose to happen?  Thanks in advance!

**Comment by roflcoopter** [comment]
An ONVIF component would be awesome, it is on my roadmap
